packages/blade/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './components';
```

Comment:
I think we can remove this right? we're not using it anymore

Reply:
yes